etc/Makefile.am

```diff
@@ -1,2 +1,2 @@
-SUBDIRS = zfs sudoers.d $(ZFS_INIT_SYSTEMD) $(ZFS_INIT_SYSV) $(ZFS_MODULE_LOAD)
+SUBDIRS = init.d zfs sudoers.d $(ZFS_INIT_SYSTEMD) $(ZFS_INIT_SYSV) $(ZFS_MODULE_LOAD)
```

Comment:
I believe `init.d` _should_ already be included here by `$(ZFS_INIT_SYSV)` as long as `--enable-sysvinit` was specified.  Everything should be included by `make dist` since they're both listed in the `DIST_SUBDIRS` target.

Reply:
@tonyhutter This was your change. Any thoughts? I agree with Brian here. See config/user-sysvinit.m4. I think this can be removed. I'm pushing a new version of the PR with it removed.

Reply:
Correct, you've structured your patch in a way where it's not needed.

Reply:
Hmm... maybe not.  I'm seeing no build failures when I manually build your patches on top of zfs-0.8-release on Centos 6, but the build failures on buildbot look like what I was seeing before I added in the `SUBDIRS = init.d` line.  Try adding it back in and see if the build failures go away.

Reply:
It looks like the fix here is going to need to be a little more involved.  We're going to need to only build the `contrib/initramfs` directory when `--enable_sysvinit` is set, and hence we know the `zfs-functions` files were built.  Adding, the `SUBDIRS = init.d` effectively works around this by always building them which isn't ideal.

Reply:
That doesn’t sound right either. initramfs doesn’t require sysvinit. We will have to find some solution. I’ll take a look. zfs-functions may need to be broken out into a separate directory or something.

Reply:
So there is already an `etc/zfs` directory. It's not obvious why `zfs-functions` isn't in there. I suspect it is historical. I moved it. How does this version work?

I also found that there is duplicated code in the initramfs from before `zfs-functions` was substituted, so I removed that.